doc/api/cli.md

```diff
@@ -627,6 +693,13 @@ Node.js options that are allowed are:
 - `--openssl-config`
 - `--pending-deprecation`
 - `--redirect-warnings`
+- `--diagnostic-report-directory`
```

Comment:
Maybe all these lines should be placed before `--enable-fips` line alphabetically?